### ISSUE TITLE
CodeFix for Assert with null message causes ambiguous code.

### ISSUE THREAD

**Issue by manfred-brands** [nunit/nunit.analyzers#700]
```
Assert.AreEqual(2d, 3d, null);
```
would generate
```
Assert.That(3d, Is.EqualTo(2d), null);
```
which does not compile
```
The call is ambiguous between the following methods or properties:
'Assert.That<TActual>(TActual, IResolveConstraint, FormattableString,
string, string)' and 'Assert.That<TActual>(TActual, IResolveConstraint,
Func<string>, string, string)'
```

**Comment by manfred-brands** [comment]
The same holds if an tolerance value is supplied:
````
Assert.AreEqual(2d, 3d, 0.1, null);
```

Which results in:
```
Assert.That(3d, Is.EqualTo(2d).Within(0.1), null);
```

**Comment by manfred-brands** [comment]
Fixed by #698
